crates/testing/src/spinning_task.rs

```diff
@@ -217,7 +228,7 @@ where
                                     self.last_decided_leaf.clone(),
                                     TestInstanceState::new(self.async_delay_config.clone()),
                                     None,
-                                    view_number,
+                                    read_storage.last_actioned_view().await,
```

Comment:
Is this equivalent to what we do in the sequencer? Do we use the last actioned view or something else?

Reply:
This should be equivalent,  the sequencer only has access to storage and has a very similar function to get the view number